### PR TITLE
{tools}[GCC/5.4.0-2.26] Singularity v2.3.1

### PR DESCRIPTION
**Pull request by frenchwr**
Adding most recent version of Singularity

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f1838e4fd146b3afaf19a306eba35409 for a full test report.

File: easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+preconfigopts = './autogen.sh && '
+
+sanity_check_paths = {
+    'files': ['bin/run-singularity', 'bin/singularity', 'etc/singularity/singularity.conf',
+              'libexec/singularity/bin/action-suid'],
+    'dirs': ['etc', 'libexec/singularity'],
+}
+
+# next steps after instalations
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb]: I know you didn't write this @frenchwr but could you fix the spelling of 'installation' here? I'm finding it painful...

**Comment by frenchwr** [comment]
@verdurin done!

File: easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb [hunk 2]
```diff
@@ -22,7 +22,7 @@ sanity_check_paths = {
     'dirs': ['etc', 'libexec/singularity'],
 }
 
-# next steps after instalations
+# next steps after installation
 # INSTALATION_PATH=your_instalation_path
```
Comment (verdurin) [inline easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb]: Thanks @frenchwr and I'm a little embarrassed even to ask this, but could you fix the other misspellings of 'installation' in this comment too?

Comment (frenchwr) [inline easybuild/easyconfigs/s/Singularity/Singularity-2.3.1-GCC-5.4.0-2.26.eb]: No problem. I should have caught those before. At least the previous author was consistent!

**Comment by verdurin** [comment]
Thanks for your patience, @frenchwr - merging.

**Pull request merged**